src/fides/service/dataset/dataset_config_service.py

```diff
@@ -146,28 +147,40 @@ def validate_dataset_config(
 
     def get_dataset_reachability(
```

Comment:
This is the main change. We want to make sure we safely catch any validation exceptions, and
1) raise them to the user if they are for the dataset we are testing
2) safely ignore them if they are for datasets on a different connection than the one we're testing